RELEASE.rst

```diff
@@ -0,0 +1,28 @@
+Release
+=======
+
+This project is released on PyPI as `coveralls`_.
+
+To cut a new release, ensure the latest master passes all tests. Then, create a release commit:
+
+1. Update the :code:`CHANGELOG.rst` with the new version.
+2. Bump the version number in :code:`setup.py`.
+3. Tag that commit with the version number (:code:`git tag x.y.z`).
```

Comment:
Good idea.

So far, this PR just describes the process that had been followed before we took over. Let's collect all our ideas for changing the process going forward. Some things that come to mind:

* The `__version__` thing you mentioned.
* Creating eggs (see the NOTE in the new RELEASE.rst)
* Doing all this on CI, maybe kicked off by a tag

Any other thoughts?